cli/pack/app_dir.go

```diff
@@ -35,7 +35,16 @@ func initAppDir(appDirPath string, projectCtx *project.ProjectCtx) error {
 				panic(err)
 			}
 
-			return relPath == ".rocks" || strings.HasPrefix(relPath, ".rocks/")
+			if relPath == ".rocks" || strings.HasPrefix(relPath, ".rocks/") {
+				return true
+			}
+
+			if _, err := os.Open(src); err != nil {
+				log.Warnf("Failed to copy: %s", err)
```

Comment:
`Failed to copy` what?

Reply:
All `os` errors contain filepath, so it will be printed.